db/schema.rb

```diff
@@ -22,6 +22,7 @@
     t.integer "quantity"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "status"
```

Comment:
have the status be determined by "quantity > 0" ? have that effect our order status?